Felienne/hedy
# Allow equality operator to compare lists starting from level 14 #1813

**Please fill out this template by replacing all content between < >**

< Type _list_ was not allowed to be used with the equality operator. With this change, lists can be compared for equality starting from level 14. >

**Description**

< Add support for lists in equality comparisons starting from level 14 > Use present tense without a subject to describe your PR, for example: "Adds translations of levels 1 to 12 to Polish"

**Fixes < #1813  >**

< Always link the number of the issue or of the discussion that your pr concerns. >
Notable exception: adding content may be done without an accompanying issue 

**How to test**

Run Hedy locally and navigate to level 13. Run the following program and ensure that you get an error message saying what is wrong and giving suggestions how to change the code:
```
a = 1, 2
b = 1, 2
if a is b
    print 'yes'
else
    print 'no'
    
```
 Now navigate to level 14. The following program should not yield errors:
```
a = 1, 2
b = 1, 2
if a == b
    print 'yes'
else
    print 'no'
```

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
